### PR TITLE
Add match for © 2021 Qualcomm Innovation Center, Inc

### PR DESCRIPTION
**Pull request by aaronkjones**
Tested with:
`docker run -it --rm -v ${PWD}:/workspace localhost:5000/repolinter:v0.11.1 --rulesetFile /workspace/repolint.json -g https://github.com/quic/gunyah-hypervisor.git`

### CODE REVIEW

File: repolint.json [hunk 1]
```diff
@@ -4,316 +4,316 @@
...
+              "babel.config.js",
+              "build/*",
+              "jest.config.js",
+              "node_modules/*",
+              "types/*"
+            ]
+          },
+          "lineCount": 40,
+          "patterns": [
+            "Copyright.*Qualcomm Innovation Center, Inc|© 2021 Qualcomm Innovation Center, Inc",
```
Comment (craigez) [inline repolint.json:160]: Why include the year? Won't that cause issues in the future?

Suggest instead:

`© .*Qualcomm`

**Review by craigez — CHANGES_REQUESTED** [review]

File: repolint.json (hunk 1 above)
Comment (aaronkjones) [reply repolint.json:160]: Thanks Craig. I had done that in testing but forgot to commit it.

**Comment by aaronkjones** [comment]
Tested again w/ another repo:

`docker run -it --rm -v ${PWD}:/workspace localhost:5000/repolinter:v0.11.1 --rulesetFile /workspace/repolint.json -g https://github.com/quic/gunyah-c-runtime.git`

**Comment by craigez** [comment]
Not blocking, but your editor and whoever last edited don't have the same config. Lots of whitespace change.

**Review by craigez — APPROVED** [review]

**Comment by aaronkjones** [comment]
It's using [jsonlint](https://github.com/zaach/jsonlint) with the default rule set. Do I need to use something different?

File: repolint.json [hunk 2]
```diff
@@ -4,316 +4,316 @@
...
+              "babel.config.js",
+              "build/*",
+              "jest.config.js",
+              "node_modules/*",
+              "types/*"
+            ]
+          },
+          "lineCount": 40,
+          "patterns": [
+            "Copyright.*Qualcomm Innovation Center, Inc|©.*Qualcomm Innovation Center, Inc",
```
Comment (mynameistechno) [inline repolint.json:160]: Like I mentioned to @aaronkjones on Friday, surprised this works without parens. E.g. is it not ORing the 'c' and the '©'? Did you test it works on both the cases and also fails when expected? Thanks

**Comment by mynameistechno** [comment]
> Not blocking, but your editor and whoever last edited don't have the same config. Lots of whitespace change.

Looks like it was setup with tabs or 4 spaces or something. Yours looks to be 2 spaces, which is probably better and the default for JavaScript formatting I think, so JSON probably the same.

File: repolint.json (hunk 2 above)
Comment (aaronkjones) [reply repolint.json:160]: I used https://regexr.sddemo.oks.drekar.qualcomm.com to test it. It matches the entire expression, not only the glyph
image

Comment (mynameistechno) [reply repolint.json:160]: Cool, thanks.

Neat app.

**Pull request merged**